lib/ASTGen/Sources/ASTGen/ASTGen.swift

```diff
@@ -4,7 +4,7 @@ import CBasicBridging
 // Needed to use SyntaxTransformVisitor's visit method.
 @_spi(SyntaxTransformVisitor)
 // Needed to use BumpPtrAllocator
-@_spi(RawSyntax)
+@_spi(BumpPtrAllocator) //updated from RawSyntax
```

Comment:
I don't think this comment carries much weight. The `_spi(RawSyntax)` import was just a temporary measure.

Reply:
okay I removed the comment, and made a new commit. please check again. thank you so much for being so patient with me.